wp-content/themes/goonj-crm/functions.php

```diff
@@ -61,3 +61,7 @@ function goonj_login_form_validation_errors( $string ) {
 
     return $string;
 }
+function goonj_custom_reset_password_form() {
+    include get_template_directory() . '/templates/password-reset.php';
+}
+add_action('login_form_rp', 'goonj_custom_reset_password_form');
```

Comment:
Notice the patter the `add_action` comes before the function defined. This is a convention we are trying to follow:
```suggestion
add_action('login_form_rp', 'goonj_custom_reset_password_form');
function goonj_custom_reset_password_form() {
    include get_template_directory() . '/templates/password-reset.php';
}
```

Reply:
can we use [`get_template_part`](https://developer.wordpress.org/reference/functions/get_template_part/)